docs/UserGuide.md

```diff
@@ -169,8 +169,8 @@ A client is either a `buyer`, or `seller` based on whether he has a property or
 
 Examples:
 
-- `add n/Betsy Crowe e/betsycrowe@example.com a/Newgate Prison p/1234567 pf/West, 1-room, $100000, $200000 i/living.png:living room`
-- `add n/John Doe p/98765432 e/johnd@example.com a/John street block 123 #01-01, pr/East, John street block 123 #01-01, 2-room, $200000`
+- `add n/Betsy Crowe e/betsycrowe@example.com a/Newgate Prison p/1234567 pf/West, 1-room, $100000, $200000`
+- `add n/John Doe p/98765432 e/johnd@example.com a/John street block 123 #01-01, pr/East, John street block 123 #01-01, 2-room, $200000 i/living.png:living room`
```

Comment:
Maybe it might be better to put `i/living.png:living room`  in line `172` instead of line`173`? It would be easier for testers to test to see if they get the same output as the screenshot since they would otherwise have to add a image in the same folder in order to test the command.

Reply:
I changed it because it does not make sense for the buyer (Betsy Crowe) to have a picture of a living room. Hence I put it under the seller (John Doe). Additionally, I added an extra example on top for inputting user profile picture (for buyers) on line 110.
```- Image must be specified in the following format `i/FILEPATH:DESCRIPTION`. Make sure your file path is from the directory the jar file is run. Description is optional and can be omitted. [exp. `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 pr/north,123 Street,1-room,$300000 i/JohnDoe.jpg:Profile Picture of John Doe`]```